cfconfiguration/cfconfiguration.go

```diff
@@ -4,31 +4,48 @@ import (
 	"io/ioutil"
 	"errors"
 	"github.com/ghodss/yaml"
+	"os"
+	"os/user"
+	"fmt"
 )
 
 type configuration struct {
+	Profile string
+	Region string
 	SpecificationURL map[string]string
 }
 
 var config configuration
+const globalConfigFile string = "/etc/.Appliscale/cftool/config.yaml"
+const userConfigFile string = "/.Appliscale/cftool/config.yaml"
 
-func GetSpecificationFileURL(region string) (string, error) {
-	err := getConfiguration()
+func GetSpecificationFileURL(configurationFilePath string) (string, error) {
+	err := getConfiguration(configurationFilePath)
```

Comment:
IMHO we should pass here URL only, not even the configuration object - and function could be called `GetSpecificationFileFromURL`.

Reply:
This also woes that in most cases you'd like to pass something called vaguely a _CLI context_ - which will mean in our case for now a logger instance and configuration file through the application flow.